src/shaders/Sky.tsx

```diff
@@ -43,7 +61,9 @@ export const Sky = forwardRef<Sky>(
         material-uniforms-mieCoefficient-value={mieCoefficient}
         material-uniforms-mieDirectionalG-value={mieDirectionalG}
         material-uniforms-rayleigh-value={rayleigh}
-        material-uniforms-sunPosition-value={sunPosition}
+        material-uniforms-sunPosition-value={[sunPositionX, sunPositionY, sunPositionZ]}
```

Comment:
Can this work without modifiyng the shader?

Reply:
Do you mean by referencing a single sunPosition prop as done previously but that combines sunPosition X, Y, Z at an earlier point?

Reply:
You are setting `inclination` and `azimuth` material uniforms, which does not exists in the shader

Reply:
Ah I must have misunderstood how that works, I'll have that changed as well